rest_framework/fields.py

```diff
@@ -417,10 +417,11 @@ def to_representation(self, value):
         Transform the *outgoing* native value into primitive data.
         """
         raise NotImplementedError(
-            '{cls}.to_representation() must be implemented.\n'
+            '{cls}.to_representation() must be implemented for field {field_name}.\n'
```

Comment:
How about we update the wording slightly and may as well drop the newline in the process:

    '{cls}.to_representation() must be implemented for field {field_name}. If you do not want the field to support write operations you might want to subclass `ReadOnlyField` instead.'

Sound okay? If no-one has any objections to that wording I'd be happy to accept this PR after this update.

Minor improvement, but yes clearly worthwhile. :smile: